app/mm-redux/client/client4.ts

```diff
@@ -2854,9 +2854,9 @@ export default class Client4 {
         );
     };
 
-    getGroups = async (filterAllowReference = false, page = 0, perPage = PER_PAGE_DEFAULT) => {
+    getGroups = async (filterAllowReference = false, page = 0, perPage = PER_PAGE_DEFAULT, since = 0) => {
```

Comment:
`0` was returning same results without `since` param. So I made it default. I will make it optional if functionality is different for this 🙂

Reply:
Ah I see you're right. I noticed backend code has
```
	if opts.Since > 0 {
		groupsQuery = groupsQuery.Where(sq.Gt{
			"g.UpdateAt": opts.Since,
		})
	} else {
		groupsQuery = groupsQuery.Where("g.DeleteAt = 0")
	}
```
Where if the parsed since is greater than 0 it does what it does otherwise it defaults to the else.